### ISSUE TITLE
SQLite IN Operator Support

### ISSUE THREAD

**Issue by Robbie-Palmer** [iafisher/isqlite#77]
When trying to use the [SQLite IN operator](https://www.sqlitetutorial.net/sqlite-in/) in a conditional clause I get the error:
```
Traceback (most recent call last):
  File "../lib/python3.9/site-packages/isqlite/database.py", line 615, in delete
    self.sql(f"DELETE FROM {quote(table)} WHERE {where}", values=values)
  File "../lib/python3.9/site-packages/isqlite/database.py", line 652, in sql
    self.cursor.execute(query, values)
sqlite3.OperationalError: near ":ids": syntax error
```

Example query:
```python
with isqlite.Database(db.database_path) as transaction:
    transaction.delete('Table', where='id in :ids', values=dict(ids=(1, 2, 3)))
```

**Comment by iafisher** [comment]
I believe this is a limitation in the underlying Python `sqlite3` library itself that requires you to individually interpolate each element of the tuple. For instance, running something like `conn.execute("DELETE FROM table WHERE id IN ids", {"ids": (1, 2, 3)})` directly in `sqlite3` doesn't work for me. [This Stack Overflow question](https://stackoverflow.com/questions/1309989/parameter-substitution-for-a-sqlite-in-clause) documents the problem and has a workaround, although it's not pretty.

**Comment by Robbie-Palmer** [comment]
Thanks @iafisher! The below works for me, allowing me to avoid a loop doing individual deletions, but as you said it's not super intuitive to look at
```python
ids = (1, 2, 3)
with isqlite.Database(db.database_path) as transaction:
    id_markers = ','.join('?' * len(ids))
    transaction.delete('Table', where='id in ({id_markers})', values=row_ids
```

I'm wondering if a function like `delete_many_by_pk` fits in with your API design? Providing an intuitive way to efficiently delete many rows

**Comment by iafisher** [comment]
I like the idea of a `delete_many_by_pk` method. It's certainly better than the workaround. Note that even though your code is obviously safe, in general you should never directly interpolate values into the `where` clause as they will not be escaped. See the [Security](https://isqlite.readthedocs.io/en/stable/security.html) guide for details.

**Comment by iafisher** [comment]
I have added this method and released it in version 1.5.0.
